### PR TITLE
Second attempt at SBAT doc tweaks

### PR DESCRIPTION
**Pull request by steve-mcintyre**

### CODE REVIEW

File: SBAT.md [hunk 1]
```diff
@@ -73,20 +73,22 @@ number are required to include fixes for any public or pre-disclosed
...
+installed on the same system and would not be subject to the other
+product's product-specific minimum generation number. However, both of
+those components will need to meet the global minimum generation
+number for that component. A very likely scenario would be that a
+product is shipped with an incomplete fix required for a specific
+minimum generation number, but is labeled with that number. Rather
+than having the entire industry that uses that component re-release,
+just that product's minimum generation number would be incremented and
+that product's component re-released along with a UEFI variable update
+specifying that requirement.
```
Comment (vathpela) [inline SBAT.md:91]: Why is this whole section getting re-flowed?

File: SBAT.md [hunk 2]
```diff
@@ -151,9 +153,9 @@ the UEFI revocation variable.
 If this same Vendor C has a similar event after the global number is
 incremented, they would again set their product-specific or version-specific
 number to 1. If they have a second event on with the same component, they would
-set their product or version-specific number to 2.
+set their product- or version-specific number to 2.
```
Comment (vathpela) [inline SBAT.md:156]: This doesn't really make anything clearer to me; the dangling '-' is distracting.  If you're going to do this, I prefer just going ahead and doing "product-specific or version-specific".

File: SBAT.md [hunk 3]
```diff
@@ -110,17 +112,17 @@ a rare event, and that the generation number for the upstream code base will
...
 number.
 
-However once the global number is bumped for the next upstream CVE fix
-there is no need to carry it any longer since satisfying the check of
-the global number will also exclude any of the older product-specific
-binaries.
+However, once the global number is bumped for the next upstream CVE
+fix there will be no further need to carry that product-specific
+generation number; satisfying the check of the global number will also
+exclude any of the older product-specific binaries.
```
Comment (vathpela) [inline SBAT.md:125]: The independent clause here makes this feel like a run-on.  Why not just make it two sentences?

File: SBAT.md [hunk 4]
```diff
@@ -151,9 +153,9 @@ the UEFI revocation variable.
 If this same Vendor C has a similar event after the global number is
 incremented, they would again set their product-specific or version-specific
 number to 1. If they have a second event on with the same component, they would
-set their product or version-specific number to 2.
+set their product- or version-specific number to 2.
 
-In such an event, a vendor would set the product or product version-specific
+In such an event, a vendor would set the product- or product version-specific
```
Comment (vathpela) [inline SBAT.md:158]: Same, but also do we need the second copy of "product" there?

File: SBAT.md [hunk 5]
```diff
@@ -194,10 +196,11 @@ generation number be incremented past the latest one shown in any
 binary for that product, effectively disabling that product on UEFI
 Secure Boot enabled systems.
 
-A subset of this case are beta-release that may contain eventually
+A subset of this case would be a beta-release that may contain eventually
 abandoned, experimental, kernel code. Such releases should have their
 product-specific generation numbers incremented past the latest one
-shown in any, released or unreleased, production key signed binary.
+shown in any, released or unreleased, binary signed with a production
```
Comment (vathpela) [inline SBAT.md:202]: Maybe that first comma belongs /after/ "released"?

File: SBAT.md (hunk 1 above)
Comment (steve-mcintyre) [reply SBAT.md:91]: apologies, editor damage :-(

File: SBAT.md (hunk 2 above)
Comment (steve-mcintyre) [reply SBAT.md:156]: sure, that works too

File: SBAT.md (hunk 3 above)
Comment (steve-mcintyre) [reply SBAT.md:125]: ACK

File: SBAT.md (hunk 5 above)
Comment (steve-mcintyre) [reply SBAT.md:202]: yup

File: SBAT.md (hunk 1 above)
Comment (steve-mcintyre) [reply SBAT.md:91]: The original para is arnitrarily indented here, but meh - undoing this change for now and re-applying meaningful changes separately. :-)

**Comment by steve-mcintyre** [comment]
This should be better I hope! :-)

**Comment by vathpela** [comment]
It'd be a *whole lot* easier to review with the patches squashed together.  That aside, at the least this needs a real commit message, so we know what was going on later when we're looking at the history.

**Review by christopherco — APPROVED** [review]

File: SBAT.md [hunk 6]
```diff
@@ -73,20 +73,20 @@ number are required to include fixes for any public or pre-disclosed
 issue required for that generation. Additionally, in the event that a
 bypass only manifests in a specific product's component, vendors may
 ask for a product-specific generation number to be published for one
-of their products components. This avoids triggering an industry wide
+of their product's components. This avoids triggering an industry wide
```
Comment (jsetje) [inline SBAT.md:76]: I think that was supposed to be plural, so either way would work.

**Review by jsetje — APPROVED** [review]
Thank you for the clean up

**Pull request merged**